sdk/identity/identity/src/credentials/defaultAzureCredential.ts

```diff
@@ -242,8 +243,34 @@ export class DefaultAzureCredential extends ChainedTokenCredential {
       createDefaultAzureDeveloperCliCredential,
     ];
 
-    // DefaultCredential constructors should not throw, instead throwing on getToken() which is handled by ChainedTokenCredential.
+    // If AZURE_TOKEN_CREDENTIALS is set, use it to determine which credentials to use.
+    // The value of AZURE_TOKEN_CREDENTIALS should be either "development" or "production".
+    if (process.env.AZURE_TOKEN_CREDENTIALS) {
```

Comment:
Use a variable for `process.env.AZURE_TOKEN_CREDENTIALS` instead for cleaner code

Reply:
done